Makefile

```diff
@@ -66,13 +66,25 @@ reset-net: check-deploy-env
 		-e ansible_python_interpreter=/usr/bin/python3 \
 		reset-net.yml
 
-mnesia_backup:
+mnesia_snapshot:
 	cd ansible && ansible-playbook \
 		--limit="tag_role_aenode:&tag_env_$(BACKUP_ENV)" \
-		-e ansible_python_interpreter=/usr/bin/python3 \
+		-e ansible_python_interpreter=/var/venv/bin/python \
 		-e download_dir=$(BACKUP_DIR) \
 		-e backup_suffix=$(BACKUP_SUFFIX) \
-		mnesia_backup.yml
+		-e db_version=$(BACKUP_DB_VERSION) \
+		-e env=$(BACKUP_ENV) \
+		mnesia_snapshot.yml
+
+mnesia_snapshot_restore:
```

Comment:
if this is not working, let's remove it ?

Reply:
Not sure what you arę talking here that is not working

Reply:
Not sure what you arę talking here that is not working